inc/gtest/gtest_1.8.1.bb

```diff
@@ -0,0 +1,26 @@
+#
+# This recipe builds googletest.
+#
+
+SUMMARY = "googletest"
```

Comment:
Did you add a unit test for your function?  I don't see it?  There should be a unit test that runs as part of the gtests.

Reply:
The unit test is added.

Reply:
Unit test is added.